packages/@guardian/source-react-components-development-kitchen/components/divider/Divider.stories.tsx

```diff
@@ -64,3 +65,13 @@ TextDivider.args = {
 	displayText: 'I am centred',
 };
 asChromaticStory(TextDivider);
+
+// *****************************************************************************
+
+export const CssOverridesDivider = Template.bind({});
+CssOverridesDivider.args = {
+	cssOverrides: css`
+		background: red;
+	`,
+};
+asChromaticStory(CssOverridesDivider);
```

Comment:
Up to this point, we use Chromatic stories to demonstrate supported variants of each component, rather than a TDD-for-UI. As such, I don't think we need a specific Chromatic story for `cssOverrides`.

Reply:
good point, removal incoming!